.ci/jenkins_job_build.sh

```diff
@@ -200,7 +200,8 @@ test_snap_build() {
 	if [[ "${ID}" == "ubuntu" && "$(uname -m)" != "x86_64" ]]; then
 		echo "Test snap build"
 		sudo apt install -y snapcraft
-		pushd ${cidir}/../../kata-containers
+		[ ! -d "${kata_repo_dir}" ] && go get -d "${kata_repo}" || true
```

Comment:
Do we enforce `GO111MODULE=auto` before calling `go get -d ...`?  If so, we're fine.  If not, we should, otherwise it'll be downloaded in a different place than the one expected.

Reply:
do you mean change it to "sudo -E GO111MODULE=auto go get -d "${kata_repo}"?

Reply:
No, nevermind.  I just double-checked and on line 167 we do:
```
source "${cidir}/lib.sh"
```

And that does:
```
# Support Golang 1.16.x.
# By default in Golang >= 1.16 GO111MODULE is set to "on",
# some subprojects in this repo may not support "go modules",
# set GO111MODULE to "auto" to enable module-aware mode only when
# a go.mod file is present in the current directory.
export GO111MODULE="auto"
```

So, we're good to go as it's here.

Reply:
I don't think we'd ever need L. 203 and the fix only lies in L. 204 because `$cidir` is a relative path. @jongwu if you really feel more comfortable with it, we can keep it -- I guess it doesn't hurt much if the checkout already exists.

Reply:
@jongwu are you strongly in favour of keeping the `go get`?

Reply:
keep it will not bring more overhead as we do check before "go get". Just in case the kata_repo is gone sometimes.